### PR TITLE
Open threads from last PR

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Auto-close the issue?

Closes threads:
[repos from tag](https://github.com/tasitlabs/TasitSDK/pull/151#discussion_r250260219)
[update truffle](https://github.com/tasitlabs/TasitSDK/pull/147#discussion_r249877923)

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: packages/tasit-contracts/decentraland/scripts/install.sh [hunk 1]
```diff
@@ -8,17 +8,43 @@ then
...
+then
+    rm -rf $REPO_DIR;
+    git clone https://github.com/decentraland/$REPO.git $REPO_DIR;
+    npm i --prefix $REPO_DIR;
+fi
+
+
+# land
+REPO="land"
+TAG="deploy/2018-08-31"
```
Comment (pcowgill) [inline packages/tasit-contracts/decentraland/scripts/install.sh]: I know I said tag before, but I'm wondering if the hash of the commit itself would be better, since tags can be changed to point to different commits

Comment (pcowgill) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: Either way this is a big improvement already

**Comment by pcowgill** [comment]
@marcelomorgado Do you agree that it appears CircleCI didn't run for this branch?

**Comment by marcelomorgado** [comment]
> @marcelomorgado Do you agree that it appears CircleCI didn't run for this branch?

It runs on my CircleCI I'm not sure but seems that always occurs on PR creation. I'll do another commit soon and then I think that it will run.

File: packages/tasit-contracts/decentraland/scripts/install.sh (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: OK, I'll commit this change in a few minutes.

**Comment by pcowgill** [comment]
> It runs on my CircleCI I'm not sure but seems that always occurs on PR creation. I'll do another commit soon and then I think that it will run.

Interesting, yeah it did run. We'll have to look into why that is at some point. Not the top priority though.

File: packages/tasit-contracts/decentraland/scripts/install.sh (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: I'm resolving this

**Comment by marcelomorgado** [comment]
> > It runs on my CircleCI I'm not sure but seems that always occurs on PR creation. I'll do another commit soon and then I think that it will run.
> 
> Interesting, yeah it did run. We'll have to look into why that is at some point. Not the top priority though.

https://github.com/tasitlabs/TasitSDK/issues/158
